src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using Xunit;
+using System.Linq;
```

Comment:
Correct, adding Linq allows to use Linq extensions while debugging and stepping in this code.